blog/2025/keycloak-kubecon25-japan-announce.adoc

```diff
@@ -0,0 +1,40 @@
+:title: Register now for KubeCon Japan in June
+:date: 2025-03-24
+:publish: true
+:author: Alexander Schwartz
+:preview: keycloak-kubecon-tokyo-2025.png
+:summary: This year is the first time there is a KubeCon in Japan, and the Keycloak project is excited to be part of it! Register today to join us on June 16-17 2025 in Tokyo, Japan for this exciting event.
+
+This year is the first time there is a KubeCon in Japan, and the Keycloak project is excited to be part of it! Join us on June 16-17 2025 in Tokyo, Japan for this exciting event.
+
+https://events.linuxfoundation.org/kubecon-cloudnativecon-japan/register/[Register today to get tickets for the standard rate.]
+
+Keycloak has a powerful community in Japan, and we have received several contributions in the past. There will be two talks about Keycloak (see below).
+
+A few days before, we will be hosting https://www.keycloak.org/2025/03/keyconf-25-japan-call-for-papers[KeyConf Japan].
```

Comment:
The event name "KeyConf Japan" will be renamed. Considering this point, I think there are two options:
1. Wait for the renaming and post the article.
2. Post the article without mentioning "KeyConf Japan".

Reply:
IMO, to attract a lot of people to attend KubeCon Japan, option 2 might be good.

Reply:
Ok, thank you for the feedback. I'm removing KeyConf